pageserver/src/metrics.rs

```diff
@@ -1576,10 +1580,14 @@ impl<'a, 'c> BasebackupQueryTimeOngoingRecording<'a, 'c> {
                 elapsed
             }
         };
-        let metric = if res.is_ok() {
-            &self.parent.ok
-        } else {
-            &self.parent.error
+        let metric = match res {
+            Ok(_) => &self.parent.ok,
+            Err(QueryError::Disconnected(ConnectionError::Io(io_error)))
+                if is_expected_io_error(io_error) =>
+            {
+                &self.parent.client_error
+            }
+            Err(_) => &self.parent.error,
```

Comment:
Some of the other QueryError variants are also client errors, or just part of normal operations (e.g. Reconnect).

What to do about them?

Reply:
We don't categorize them as client error for now. Currently we use the same criteria as `log_query_error`, info logs <-> client error. I added a comment in the file so that if anyone wants to re-leveling the logs, they will re-categorize the metrics bucket.

Reply:
thanks!